src/main/java/app/model/Label.java

```diff
@@ -0,0 +1,36 @@
+package app.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.JoinTable;
+import jakarta.persistence.ManyToMany;
+import jakarta.persistence.Table;
+import java.util.List;
+import lombok.Data;
+import org.hibernate.annotations.SoftDelete;
+
+@Entity
+@Table(name = "labels")
+@SoftDelete(columnName = "is_deleted")
+@Data
+public class Label {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private String color;
+
+    @ManyToMany
```

Comment:
Do you have to have this, or does Task entity should have a list of labels?

Reply:
resolved by swapping join column and reversejoincolumn